app/views/games/finished.html.erb

```diff
@@ -9,7 +9,7 @@
       </div>
     </div>
     <div class="too_late">
-      <% if @game.players.include?(current_player) && current_player.won == false %>
+      <% if @game.players.include?(current_player) && current_player.won %>
```

Comment:
While we're at it, shouldn't we extract this logic (”is the current player playing in the current game and has he won?”) to something like a `Game#winner_is?` method? Or are those two checks unrelated?